docs/modules/system preparation.rst

```diff
@@ -0,0 +1,309 @@
+.. _system_preparation:
+
+Preparing systems for constant-pH simulation
```

Comment:
Is it weird to document a procedure we will be immediately retiring?
Or do we plan to actually release a version with the Amber `cpin` interface?